graphql/src/Mu/GraphQL/Server.hs

```diff
@@ -57,6 +56,7 @@ import           Network.Wai.Handler.Warp         (Port, Settings, run, runSetti
 import qualified Network.Wai.Handler.WebSockets   as WS
 import qualified Network.WebSockets               as WS
 
+import           Mu.GraphQL.Quasi.LostParser      (parseDoc)
```

Comment:
Is my "lost parser" still needed with this new parser? 🤔

Reply:
Now it's more like "function which runs the parser and juggles a bit with the result so it pleases what we need", but I decided to keep the name xD